### DISCUSSION TITLE
comp. mod / a base remix- mod

### DISCUSSION THREAD

**Discussion by Ogwy** [xoxor4d/p2-rtx#3] [Q&A]
What's the difference between the comp.mod and the base remix.mod?

**Comment by xoxor4d** [comment]
> What's the difference between the comp.mod and the base remix.mod?

The compatibility mod is required to make the game work with rtx remix in the first place.

The remix mod is an actual mod created using the remix toolset. It makes the portals work, handles some emissive textures and adds lights to some scenes. It is not required to get the pathtracing to work. It is meant as a base for modders or people that just want to take a quick look without waiting for a proper remix mod that remasters all the textures, meshes etc.
